core/templates/pages/exploration-editor-page/changes-in-human-readable-form/changes-in-human-readable-form.component.html

```diff
@@ -1,153 +1,153 @@
                     <p>Value:
                       <span> </span>
-                      <ng-content ng-repeat="input in rule.inputs">
-                        <[ $last ? input : (input + ',')]>
-                      </ng-content>
+                      <ng-container *ngFor="let input of rule.inputs">
+                        {{  last ? input : (input + ',') }}
+                      </ng-container>
                     </p>
                   </li>
                 </ol>
               </div>
             </div>
 
-            <div ng-if="lostChange.getRelativeChangeToGroups() === 'deleted'" class="state-edit-desc">
+            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'deleted'" class="state-edit-desc">
               Deleted answer group
             </div>
           </div>
 
-          <div ng-switch-when="default_outcome">
-            <div ng-if="lostChange.getRelativeChangeToGroups() === 'added'" class="state-edit-desc default-outcome">
+          <div *ngSwitchCase="'default_outcome'">
+            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'added'" class="state-edit-desc default-outcome">
               Added default outcome:
               <p class="sub-edit">
-                <i>Destination: </i><[lostChange.newValue.dest]>
+                <i>Destination: </i>{{ lostChange.newValue.dest }}
               </p>
               <div class="sub-edit">
                 <i>Feedback: </i>
                 <div class="feedback">
-                  <[lostChange.newValue.feedback.html]>
+                  {{ lostChange.newValue.feedback.html }}
                 </div>
               </div>
             </div>
 
-            <div ng-if="lostChange.getRelativeChangeToGroups() === 'edited'" class="state-edit-desc default-outcome">
+            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'edited'" class="state-edit-desc default-outcome">
               Edited default outcome:
-              <p class="sub-edit" ng-if="!isDestEqual()">
-                <i>Destination: </i><[lostChange.newValue.dest]>
+              <p class="sub-edit" *ngIf="!isDestEqual()">
+                <i>Destination: </i>{{ lostChange.newValue.dest }}
               </p>
 
-              <div class="sub-edit" ng-if="lostChange.isFeedbackEqual()">
+              <div class="sub-edit" *ngIf="lostChange.isFeedbackEqual()">
                 <i>Feedback: </i>
                 <div class="feedback">
-                  <[lostChange.newValue.feedback.html]>
+                  {{ lostChange.newValue.feedback.html }}
                 </div>
               </div>
             </div>
 
-            <div ng-if="lostChange.getRelativeChangeToGroups() === 'deleted'" class="state-edit-desc">
+            <div *ngIf="lostChange.getRelativeChangeToGroups() === 'deleted'" class="state-edit-desc">
               Deleted default outcome
             </div>
           </div>
-        </ng-content>
+        </ng-container>
```

Comment:
Expected?

Reply:
`[ngSwitch]` is not working with ng-content.
(Another alternative would be creating a new div.)